portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,226 @@
-  width: 650px;
+  padding: 20px;
+  max-width: 1000px;
+  background-color: white;
+  font-size: 20px;
+}
+
+h3 {
+  text-shadow: 1px 1px 3px #ccc;
+}
+
+.info-bar {
+  border-right: 1px solid #ccc;
+}
+
+#name {
+  font-weight: bold;
+  margin: 15px 0px;
 }
 
-#greeting-container {
+#location i {
+  font-size: 24px;
+}
+#location span {
+  font-size: 18px;
+  margin: 20px 0px;
+}
+
+#links {
   margin-top: 20px;
 }
+#links a {
+  display: block;
+  margin-top: 10px;
+  color: #7c7b8a;
+}
+#links a i {
+  font-size: 30px;
+}
+
+.fa-linkedin-square {
+  color: #308ECA;
+}
+
+.fa-github {
+  color: black;
+}
+
+.fa-envelope-o {
+  margin-top: 20px;
+  color: #c44a41;
+}
+
+/* Avatar top image, hidden by default. */
+.img-top {
+  display: none;
+}
+#avatar img {
+  max-height: 120px;
+}
+/* On hover, only displays top image. */
+#avatar:hover img {
+  display: none;
+}
+#avatar:hover .img-top {
+  display: inline;
+}
+
+.top-nav {
+  padding: 4px;
+}
+
+/* index.html */
+
+.fa-hand-o-right {
+  margin-left: 10px;
+}
+
+#my-name {
+  font-weight: 800;
+  font-size: 30px;
+  color: #3aa7d6;
+  text-shadow: 1px 2px 4px #ccc;
+}
+#my-name:hover {
+  color: #a7c1fa;
+}
+
+.text-bold {
+  font-weight: 700;
+  font-size: 22px;
+  color: #607bd1;
+}
+
+.text-red {
+  color: #b34949;
+}
+
+#talents-title:hover {
+  cursor: pointer;
+  color: #7c7b8a;
+}
+
+#fun-title {
+  margin-top: 50px;
+}
+
+#fun-div {
+  text-align: center;
+}
+
+#fun-div button {
+  border-radius: 10px;
+  color: white;
+  width: 100%;
+  padding: 8px;
+  margin-bottom: 10px;
+  background-color: #ed7979;
+  border: 2px solid #ed7979;
+  box-shadow: 5px 5px 8px #ab9da8;
+}
+#fun-div button:hover {
+  background-color: white;
+  color: black;
+}
+
+.img-fact {
+  max-width: 80px;
+}
+
+/* projects.html */
+
+#projects-div {
+  margin-bottom: 30px;
+}
+
+.subtitle {
+  margin: 20px;
+  text-align: center;
+}
+
+/* Dropdown button. */
+.dropdown-click {
+  background-color: rgb(159, 204, 234);
+  color: rgb(54, 53, 53);
+  font-size: 22px;
+  font-weight: 600;
+  font-family: "Times New Roman", Times, serif;
+  padding: 8px;
+  cursor: pointer;
+}
+.dropdown-click:hover, .dropdown-click:focus {
+  background-color: #6db4e2;
+}
+
+/* Dropdown content, hidden by default. */
+.dropdown-content {
+  display: none;
+  position: relative;
+}
+/* Shows previously hidden content. */
+.show {
+  display:block;
+}
+
+/* gallery.html */
+
+/* Side nav bar, contained in the gallery. */
+.side-nav {
+  position: fixed;
+  padding: 20px;
+}
+
+/* Gallery contents, displayed in grid layout. */
+.tab-content {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 4px;
+  border: none;
+}
+.column {
+  flex: 10%;
+  max-width: 25%;
+  padding: 0 4px;
+}
+.column img {
+  margin-top: 8px;
+  margin-bottom: 8px;
+  vertical-align: middle;
+  max-width: 100%;
+}
+
+/* Container for every single photo in the gallery. */
+.photo {
+  position: relative;
+}
+/* Sets opacaity when hovered. */
```

Comment:
I would move this comment to inside the block, since the opacity is changed only by the single line and you could always be adding more to this block as a whole.

Reply:
Got it.